honeybee_grasshopper_core/src/HB Vizualize Quick.py

```diff
@@ -0,0 +1,51 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Quickly preview any Honeybee geometry object within the Rhino scene.
+_
+Sub-faces and assigned shades will not be included in the output, allowing for
```

Comment:
This is smart. I think one other relatively fast options is to just draw the geometries including the sub-faces as polylines. This might even be faster than the face with no sub-face since depending on tolerance creating a Brep face can take a very long time.

Reply:
Hm. That's a good idea. I think both types of components have an important role to fill. It will only take me 2 minutes to add a "HB Vizualize Wireframe" component to this PR.  Let me do that now.

Reply:
I just wanted to add that this was a great suggestion. The wireframe preview is extremely fast.  It will be very useful for checking large models.